locales/en/actions.json

```diff
@@ -16,6 +16,7 @@
   "continue": "Continue",
   "restart": "Restart",
   "cancel": "Cancel",
+  "cancel_allocation": "Cancel allocation",
```

Comment:
`Cancel this allocation`? Remains consistent with the copy used below.

Reply:
Worth noting @teneightfive, on the cancel page the red button is 'Cancel move'. Probably should also be consistent in that case.

Or the text link should set the expectation that you are not cancelling by clicking the link (ie. "I need to cancel this move / allocation"

Worth catching up to talk through.

Reply:
Yep. I think we use a slight difference between button and link text. One to suggest you might go somewhere, one to suggest you'll do the final action.

Reply:
Hmm -- ok, so shall I change it it "I need to cancel this allocation"?

Reply:
Hold off for a bit on this one @barbaracassani - need to think it through as it affects other journeys.

@teneightfive Cancel this XXX and Cancel XXX doesn't do enough to suggest difference in my opinion. Let's catch up and chat through with other journeys